utils/build/docker/nodejs/express4/app.js

```diff
@@ -2,6 +2,9 @@
 
 const tracer = require("dd-trace").init({
   debug: true,
+  appsec: {
```

Comment:
why add this?
We already had some appsec related test that were working, may already be activated by environment var?

Reply:
left it by mistake